### PR TITLE
Separate commands into their own files.

### PR DESCRIPTION
**Pull request by noshidayo**
Also fix getPoints function.

### CODE REVIEW

File: behaviour/dealer.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
-from behaviour.base import Behaviour
+from behaviour._base import Behaviour
+import command
 
 class DealerBehaviour(Behaviour):
 
 	def tick(self, player, game):
-
+
```
Comment (paulryannpaqueo) [inline behaviour/dealer.py]: what is this?

Comment (noshidayo) [reply behaviour/dealer.py]: 1 line too many will remove

File: blackjack.py [hunk 2]
```diff
@@ -158,7 +132,8 @@ def start(self):
 		for p in self.players + [self.dealer]:
 			print("It's %s's turn." % p.name)
 			while p.getPoints() <= 21:
-				cmd = p.behaviour.tick(p, self)
+				cmdmod = p.behaviour.tick(p, self)
+				cmd = cmdmod.ImplementedCommand()
 				if cmd.execute(p, self):
```
Comment (flamestream) [inline blackjack.py]: Please use a more descriptive name.

File: behaviour/__init__.py [hunk 3]
```diff
@@ -5,5 +5,3 @@
 tryhard = behaviour.tryhard.TryHardBehaviour()
 dealer = behaviour.dealer.DealerBehaviour()
 human = behaviour.human.HumanBehaviour()
-
-__all__ = [tryhard, human, dealer]
```
Comment (flamestream) [inline behaviour/__init__.py]: Would be nice to use the same approach as command so there is no need to add new behaviours in this file.

**Review by flamestream — CHANGES_REQUESTED** [review]

**Review by flamestream — APPROVED** [review]

**Comment by flamestream** [comment]
👍

**Pull request merged**